Roll Command

lichmeh opened the pull request:
Created a /diceroll 1-10 command that rolls the specified amount of dice
 No args or Invalid Args defaults to 2 dice
 Anything set over 10 will cap the # of dice.

freshlogic commented:
Shouldn't we work in an option for the number of sides on the dice? I figure there's tons of prior art out there on IRC: http://www.netlingo.com/more/irccommands.php

lichmeh commented:
@freshlogic That was my original plan, but I didn't want to make it too complicated.  Do we limit them all to the same # of sides, or make it so each arg is the # of sides for a die?

Example:
/diceroll 5 10 would roll 5 dice, each with 10 sides

or 
/diceroll 5 10 would roll two dice, one with 5 sides and one with 10

freshlogic commented:
https://en.m.wikipedia.org/wiki/Dice_notation

freshlogic commented:
https://www.npmjs.com/package/roll

lichmeh commented:
Modified command to /roll instead of /diceroll
Added dependency for /roll in package.json
Updated readme file
Updated test/index.js file
Updated lib/index.js file

/roll notation supported with option -show flag to explain total
/roll defaults to 1-100
/roll --help gives usage examples

freshlogic commented on lib/index.js:
```diff
@@ -459,6 +460,49 @@ const commands = {
+	args = args.replace('-show','').trim();
+
+        if (!args) {
+            args = 'd100';
```
Don't you think most people would expect a `d6` here?

lichmeh replied:
Depends on the user and context.  When I think of "rolling the dice", I think of 2d6 over d6.

The reason I went with d% (d100) is because I think it has a much higher probability of being used day to day than a d6 or 2d6 would, and people who already know notation will probably specify it anyways if they wanted a d6.

I can change it, but if I do, I'd like to make a /random command also that defaults 1-100, as I see that getting more use.

freshlogic commented on lib/index.js:
```diff
@@ -459,6 +460,49 @@ const commands = {
+            args = 'd100';
+        }
+
+	if (args == '--help') {
```
Probably also want `-h`
